en/docs/reference/configurations/configuring-micro-gw-conf-for-kubernetes.md

```diff
@@ -0,0 +1,151 @@
+# Configuring micro-gw.conf for Kubernetes
+
+You can use environment variables when running Kubernetes to override the configurations in the `<MGW-RUNTIME-HOME>/conf/micro-gw.conf` file.
+
+- You need to create the variable name by appending the `micro-gw.conf` configuration hierarchy by using an `_` underscore to define each level of the hierarchy.
+
+## Example 1 - Defining the Listener HTTPS port with secret references
```

Comment:
Is it correct to understand that people can use secret references for non-sensitive information as well?

Reply:
You can but it is not a practice developers follow. Because then there is a burden of secret management which may not be accessible for certain devs. But no harm in using that for an example I guess.

Reply:
Ack